### PR TITLE
Update readme and namespace packages

### PR DESCRIPTION
**Pull request by Zombiefruit**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,172 +1,99 @@
...
-- api documentation 
-- simulation panel allowing:
-    - choose between different simulations 
-    - modify simulation ( no save )
-        - Automatic property panel 
-        - asset chooser
-- test panel showing the component tests, allowing filtering tests of interest
+Runs over every simulation and asserts the following:
 
+- the component can render to string (i.e. renderToString doesn't throw)
```
Comment (JoMarton) [inline README.md]: state that is for SSR

File: README.md [hunk 2]
```diff
@@ -1,172 +1,99 @@
...
+```shell
+autotools sanity --files ./components/**/*.meta.ts
 ```
-$ auto-tools website --files ./components/**/*.meta.ts
 
-```
+### A11Y
 
-#### usage dev
+Asserts that components are compatable with axe-core. Allows for varying levels of error impact (one of `minor`, `moderate`, `serious`, or `critical`).
```
Comment (JoMarton) [inline README.md]: explain the level of error impact: minor means minor and above

File: README.md [hunk 3]
```diff
@@ -1,172 +1,99 @@
...
+
 [![Build Status](https://travis-ci.org/wix-incubator/ui-autotools.svg?branch=master)](https://travis-ci.org/wix-incubator/ui-autotools)
 
 A set of tools designed to automate and improve the process of developing components.
 
-These tools rely on `*.meta.tsx` files in the project. 
+These tools rely on `*.meta.tsx` files in the project.
 
 The tools use a similiar command-line pattern:
+
```
Comment (danielstro) [inline README.md]: Maybe we should place this part in its own section (Usage, etc..)?

File: README.md [hunk 4]
```diff
@@ -1,172 +1,99 @@
...
+    - the component can render to string
+    - hydration in the client works as intended
+    - the component has no errors in <React.StrictMode />
+    - nothing was printed to the console
 - `a11y` - accessibility test:
     - checks component render result for accessibility using axe-core
+
+## WIP Tools
+
+- `eyes` - tools for generating and testing component images
```
Comment (danielstro) [inline README.md]: Maybe specify we are using eyes, although the name implies it.

**Pull request merged**